Contents/mods/!pz_debugTools/media/lua/client/LuaExplorerPanel/LuaExplorerTab.lua

```diff
@@ -59,8 +67,10 @@ function AUDLuaExplorerTab:initialise(category)
     local button
     if self.fileCategory == "ALL" then
         button = ISButton:new(0, 0, 50, self.fileList.itemheight, "+", self, self.onButtonAdd)
-    else
```

Comment:
Replaced the `else` statement with `elseif` to ensure that each button can only be set in its respective tab